Sources/Shared/Fetcher.swift

```diff
@@ -96,4 +96,11 @@ class Fetcher {
       closure()
     }
   }
+
+  static func == (lhs: Fetcher, rhs: Fetcher) -> Bool {
```

Comment:
To remove the fetcher in the manager when they are done. I thought we need it to be equatable for us to use `index(of:)`. Maybe we don't need that at all :)